packages/sdk/src/across/constants.ts

```diff
@@ -37,6 +39,7 @@ export const ADDRESSES = {
   USDC: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
 };
 
+// TODO: Deprecate this hardcoded mapping and read from RateModelStore contract via ./rateModel.ts helper methods.
```

Comment:
So this means that this PR will not change the behavior, yet, right?

Reply:
Nope

Reply:
Oh wait, yes, this PR will change the bot's behavior but not the front-end until `lpFeeCalculator.getLpFeePct` implementation is changed, see my updated comment